followthemoney/schema/Position.yaml

```diff
@@ -0,0 +1,65 @@
+Position:
+  label: "Position"
+  plural: "Positions"
+  matchable: false
+  # cf. https://www.popoloproject.com/specs/post.html
+  description: >
+    A post, role or position within an organization or body.
+    This describes a position one or more people may occupy
+    and not the occupation of the post by a specific individual at a 
+    specific point in time.
+
+    'subnationalArea' should be used to further restrict the scope of the
+    position. It should not simply represent some regional aspect of the
+    role - e.g. the constituency of a national member of parliament - when
+    their their legislative jurisdiction is nationwide.
+  featured:
+    - name
+    - organization
+    - category
+  caption:
+    - name
+  required:
+    - name
+  properties:
+    organization:
+      label: "Organization"
+      type: entity
+      reverse:
+        name: positions
+        label: Positions
+      range: Organization
+    name:
+      label: "Name"
+      type: name
+    category:
+      label: "Category"
+      type: string
```

Comment:
Not sure what the use case for `category` is, but maybe this is already covered by `Thing:keywords`?

Reply:
@pudo I was supposed to drop `category` from `Position` to synthesise it for presentation on the summary page, wasn't I?